database/db.js

```diff
@@ -1,104 +1,121 @@
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/airBnB-availability6');
-const db = mongoose.connection;
-db.on('error', console.error.bind(console, 'airBnB-availability connection error: '));
-db.once('open', function() {
-  console.log('database is open');
-});
-
-const calendarSchema = new mongoose.Schema({
-  id: Number,
-  stayId: String,
-  date: Date,
-  isAvailable: Boolean,
-  nightlyRate: Number,
-  cleaningFee: Number,
-  serviceFee: Number,
-  occupancyTaxes: Number
-
-});
-
-const Calendar = mongoose.model('Calendar', calendarSchema);
-
-const staySchema = new mongoose.Schema({
-  id: Number,
-  productId: Number,
-  minRate: Number
-});
-
-const Stay = mongoose.model('Stay', staySchema);
-
-var getMinNightlyRate = (productId, cb) => {
-  var query = Stay.where({productId: productId});
-  query.findOne( (err, stay) => {
-    if (err || stay === null || stay === undefined) {
-      //console.log('couldnt find it');
-      cb(new Error('could not find a stay with that ID'));
-    } else {
-
-      cb(null, stay);
-    }
+const {exampleData} = require('../client/src/exampleData.js');
+require('dotenv');
+
+if (process.env.NODE_ENV !== 'test') { //for testing and circleCI
```

Comment:
I might suggest mocking the db functions in your test file versus adding an if / else block to your entire file. I don't know what the best practice is 🤷‍♂️ This certainly works. I just try to keep my files from knowing they're being tested.